### PR TITLE
you can now pull the pilot out of a spacepod

### LINKED ISSUE #22309: No way to remove corpses from space pods
#### Description of issue
If someone dies while inside of a space pod there's no apparent way to remove their corpse.

#### Difference between expected and actual behavior
There could be an verb to eject a space pod's passenger from outside.

#### Steps to reproduce
Have someone die in a space pod and cry as there's no way to bring them back.

#### Server revision

Game: f71c67f5b69c1fcaa76465c747c4e5c7d4054a69 
Patches: be429b80a984e355bcb504991b007807584552c4

### PR DESCRIPTION
**Pull request by PaulRitter**
[qol]
Adds the possibility to pull the pilot out of a spacepod
closes #22309
:cl:
 * rscadd: You can now pull pilots out of their spacepods

### CODE REVIEW

**Comment by Blithering** [comment]
Any chance of a similar feature for mechs? someone passing out in a mech inevitably results in either them having to destroy it or leave it as the poor sod's tomb. I know the electropack eject button is a thing, but lets be honest roboticists who put beacons and eject buttons on mechs are as common as assistants who take the role to learn

**Comment by PaulRitter** [comment]
will look into it

**Comment by jknpj** [comment]
could just make dead occupants be ejected by default

**Comment by PaulRitter** [comment]
that'd mean that you would have no change to be pulled out if you are in a crit/knocked out slowly sliding into a crit and you'd have to wait to die. Doesn't make much sense to me

edit: that'd also mean that if you die while still moving, your body will just sit there in space while the spacepod keeps going

File: code/WorkInProgress/pomf/spacepods/spacepods.dm [hunk 1]
```diff
@@ -393,9 +393,14 @@
...
-	if(!occupants.Find(usr))
-		return ..() //Handle mousedrop T
 	var/turf/T = get_turf(over)
+	if(!occupants.Find(usr))
+		visible_message("<span class='notice'>[usr] start pulling [get_pilot().name] out of \the [src].</span>")
+		if(do_after(usr, src, 4 SECONDS))
+			move_outside(get_pilot(), T)
+			add_fingerprint(usr)
+		return
+		//return ..() //Handle mousedrop T
```
Comment (ShiftyRail) [inline code/WorkInProgress/pomf/spacepods/spacepods.dm]: This comment is useless and confusing, remove it.

Comment (PaulRitter) [reply code/WorkInProgress/pomf/spacepods/spacepods.dm]: done

**Pull request merged**